WebNetSample/Views/Home/Index.cshtml

```diff
@@ -1,9 +1,20 @@
-﻿@{
+﻿@using WebNetSample.Entity.Dtos;
+@model IEnumerable<CategoryDetailDto>
+
+@{
     ViewData["Title"] = "Home Page";
+    Layout = "_HomeLayout";
 }
 
 <div class="text-center">
     <h1 class="display-4">Welcome to Epam WEB .NET Mentoring Program</h1>
-    <h5 class="text-danger"><span class="text-dark">Mentee </span> Osman Ali YARDIM</h5>
-    <h5 class="text-warning"><span class="text-dark">Mentor </span> Dzmitry Yaniuk</h5>
+
+    @section Images {
+        @foreach (var item in Model)
+        {
+            <img src="@item.ImagePath" alt="@item.Name" style="height:10%;width:12.5%;">
```

Comment:
Just an advice: try to keep your styles declaration separately in CSS files for all elements and tags and avoid usage inline styles. The first reason is that inline style has higher priority than class, it will override classes styles and might cause some layout issues. You can take a look at [this](https://www.w3schools.com/css/css_specificity.asp) article for more information. The second reason is mixing of your approaches: usage of CSS and inline styles. Of course, there are some cases when you have to use inline styles but they occur quite rare, so, just keep that in mind ;)